lms/djangoapps/student_enrollment/enrollment.py

```diff
@@ -28,6 +28,7 @@
 initial student onboarding/enrollment process like the Careers module.
 """
 EXCLUDED_FROM_ONBOARDING = ['course-v1:code_institute+cc_101+2018_T1']
+PROGRAMS_WITH_LEARNING_SUPPORTS = settings.LEARNING_SUPPORTS_PROGRAMME_ELIGIBLE
```

Comment:
I'd be interested in discussing whether it's viable to put this info in a DB field instead.

Reply:
Definitely an option too - I'm just afraid it'd complicate DB relations and make the existing models a bit less "transparent".